people.json

```diff
@@ -1124,5 +1124,47 @@
             "Android SDK",
             "Xamarin.Forms"
         ]
+    },
+
+    "Bryan Gillis": {
+        "interests": [
+            "Python (programming language)",
+            "LaTeX",
+            "Markdown",
+            "Git",
+            "GitHub",
+            "Astrophysics",
+            "Bash (Unix shell)",
+            "Windows Subsystem for Linux",
+            "Microsoft Windows",
+            "C Sharp (programming language)",
+            "Project management",
+            "JSON",
+            "Software design",
+            "Eclipse (software)",
+            "Java (programming language)",
+            "C (programming language)",
+            "C++",
+            "Continuous integration",
+            "Linux",
+            "VS Code",
+            "HTML",
+            "Cascading Style Sheets",
+            "JavaScript",
+            "Web application",
+            "Web development",
+            "HPC",
+            "HTC",
+            "VirtualBox",
+            "Microsoft Azure",
+            "Amazon Web Services",
+            "Cloud computing",
+	    "Unit Testing",
+	    "Integration Testing",
```

Comment:
The test doesn't (ironically) find 'Integration Testing'. In this case it looks like it fails on capitalisation of Testing (see the [article](https://en.wikipedia.org/wiki/Integration_testing)). Could you change to a lower case t?

Reply:
Done. Though oddly, the line above it shows "Unit Testing" with a capital T, while the Wikipedia article for that too has a lowercase t. Let's see if this works, first, and then maybe puzzle that out